lib/turn.rb

```diff
@@ -0,0 +1,94 @@
+class Turn
+  attr_reader :player1, :player2, :spoils_of_war, :winner
+  def initialize(player1, player2)
+    
+    @player1 = player1
+    @player2 = player2
+    @spoils_of_war = []
+    @winner = nil
+  end
+
+  def type 
+    if player1.deck.rank_of_card_at(0) != player2.deck.rank_of_card_at(0)
+    :basic
```

Comment:
this applies to several places in this file; make sure the code within your `if` branches is indented properly.
```
if player1.deck.rank_of_card_at(0) != player2.deck.rank_of_card_at(0)
  :basic
elsif [other condition]
  :war
else
  :mutually_assured_destruction
end
```